### PR TITLE
feat: add token's icon & empty message & placeholder

### PR DESCRIPTION
**Pull request by chef-eric**
---

## PR-Codex overview
The focus of this PR is to update the `MultiSelect` component in the `@pancakeswap/uikit` package.

### Detailed summary
- Updated `MultiSelect` component with new features and styles
- Added `selectAllLabel` prop
- Added new `Empty` icon
- Enhanced hover effect on checkboxes
- Improved display of selected items with icons
- Added placeholder text when no items are selected

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/uikit/src/components/MultiSelect/MultiSelect.tsx [hunk 1]
```diff
@@ -271,14 +361,30 @@ export const MultiSelect = (props: IMultiSelectProps) => {
...
           {selectedItems?.map((item, idx) => (
             <React.Fragment key={item}>
               <span>{options?.find((op) => op.value === item)?.label}</span>
               {idx < selectedItems.length - 1 && <span>, </span>}
             </React.Fragment>
           ))}
-        </SelectedItemsContainer>
+        </SelectedInputItemsContainer>
+        {!selectedItems?.length ? (
+          <SelectInputPlaceholder>{props.placeholder ?? "Select something"}</SelectInputPlaceholder>
```
Comment (chef-eric) [inline packages/uikit/src/components/MultiSelect/MultiSelect.tsx:386]: add placeholder cc @ChefMomota

**Review by chefjackson — DISMISSED** [review]

**Comment by chefjackson** [comment]
@chef-eric please help resolve the conflicts

File: packages/uikit/src/components/Svg/Icons/Empty.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const Icon: React.FC<React.PropsWithChildren<SvgProps>> = (props) => {
+  return (
+    <Svg viewBox="0 0 16 5" {...props}>
+      <path
+        d="M0.5 20C0.5 9.23046 9.23045 0.5 20 0.5H60C70.7695 0.5 79.5 9.23045 79.5 20V60.0003C79.5 70.7698 70.7695 79.5003 60 79.5003H20C9.23045 79.5003 0.5 70.7699 0.5 60.0003V20Z"
+        fill="url(#pattern0_2423_223144)"
+      />
+      <path
+        d="M0.5 20C0.5 9.23046 9.23045 0.5 20 0.5H60C70.7695 0.5 79.5 9.23045 79.5 20V60.0003C79.5 70.7698 70.7695 79.5003 60 79.5003H20C9.23045 79.5003 0.5 70.7699 0.5 60.0003V20Z"
+        stroke="#E7E3EB"
```
Comment (ChefJerry) [inline packages/uikit/src/components/Svg/Icons/Empty.tsx:14]: does this Icon not support pass color props? as it fixed stroke color, how about use `currentColor`

Comment (chef-eric) [reply packages/uikit/src/components/Svg/Icons/Empty.tsx:14]: sure, this svg will replace from designer, i'll fix it in next pr

**Review by ChefJerry — APPROVED** [review]

**Pull request merged**